core/domain/feedback_services.py

```diff
@@ -406,16 +442,77 @@ def clear_feedback_message_references(user_id, exploration_id, thread_id):
         model.put()
 
 
-def add_message_to_email_buffer(author_id, exploration_id, thread_id,
-                                message_id):
+def get_all_recipient_ids(exploration_id, thread_id, author_id):
```

Comment:
Can we indicate that this is private (by adding a leading underscore)? Also, can we do the same with any other function that isn't called from outside this module?